ngrinder-controller/src/main/java/org/ngrinder/script/service/GitHubFileEntryService.java

```diff
@@ -91,7 +90,7 @@ public GitHubFileEntryService(FileEntryService fileEntryService, @Lazy ObjectMap
 	public FileEntry getOne(GHRepository ghRepository, GitHubConfig gitHubConfig, String scriptPath) {
 		String fullPath = getCheckoutDirPath(ghRepository, gitHubConfig, scriptPath);
 		if (proxy(this).isGroovyMavenProject(ghRepository, scriptPath, gitHubConfig.getBranch())) {
-			fullPath += scriptPath.substring(scriptPath.indexOf(MAVEN_PATH));
+			fullPath += scriptPath.substring(scriptPath.indexOf(JAVA));
```

Comment:
I think that it is difficult to infer what the constant `JAVA` is in this context.

Reply:
It's changed to handle in `GroovyMavenScriptHandler`

https://github.com/naver/ngrinder/pull/606/commits/8a271e48d744cc532efe317ad3560f2ef883e59a#diff-13adbe231e465545ce25ac777a738e18R305